src/unitxt/operators.py

```diff
@@ -190,7 +190,7 @@ def process(
             if value is not None:
                 if (self.process_every_value is True) and (not isinstance(value, list)):
                     raise ValueError(
-                        f"'process_every_field' == True is allowed only when all fields which have mappers, i.e., {list(self.mappers.keys())} are lists. Instance = {instance}"
+                        f"'process_every_field' == True is allowed only when for fields whose values are lists, but value of field '{key}' is '{value}'"
```

Comment:
suggesting: 
f"'process_every_field' == True is allowed only for fields whose values are lists, but value of field '{key}' is '{value}'"
(eliminating the word `when`)